week04/1268. Search Suggestions System.cpp

```diff
@@ -0,0 +1,33 @@
+// https://leetcode.com/problems/search-suggestions-system/
+// Time Complexity: O(N*logN)
+// Spcae Complexity: O(logN)
+
```

Comment:
Both are wrong.

Space complexity is clearly `O(S)` where S = Σ words

About the time complexity, definition of N is not clear.